Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -208,8 +208,10 @@ void CVolumeGC::LoadBannerFile() const
 
   GCBanner banner_file;
   std::unique_ptr<IFileSystem> file_system(CreateFileSystem(this, PARTITION_NONE));
-  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
+  if (!file_system)
+    return;
 
+  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
```

Comment:
`GetFileSize` should probably be returning a size_t anyway, not a u64

otherwise, `static_cast`